### DISCUSSION TITLE
Automatic Fiber Bundle Extraction with RecoBundles in DIPY 1.7 broken?

### DISCUSSION THREAD

**Discussion by ZubicarayG** [dipy/dipy#2937] [Tractography / Tractometry]
## Description
Automatic Fiber Bundle Extraction with RecoBundles in 1.7 appears to be broken. You can see this in the example extractions on the tutorial web page, e.g., the AF tract:

![sphx_glr_bundle_extraction_004](https://github.com/dipy/dipy/assets/119824950/27009a96-31fc-4d34-ae69-ad2d7c6683b1)

I am able to reproduce the same faulty bundle extractions running recobundles under MacOS 13.6 (Intel)

**Comment by skoudoro** [comment]
Hi @ZubicarayG,

Thank you for the feedback. 

- Recobundles is working correctly. 
- it seems to be a visualization issue. we will report it to the fury package. if you downgrade fury version, it seems to work as expected.

**Reply by ZubicarayG** [reply]
Thanks @skoudoro 

Confirming that downgrading fury to 0.8 worked!
